tests/infer/test_valid_models.py

```diff
@@ -116,6 +116,27 @@ def guide():
         assert_ok(model, guide, elbo)
 
 
+@pytest.mark.parametrize("Elbo", [
```

Comment:
I've added this test to sanity check #2288 in the presence of new tracing logic for `"_do_not_trace"`.